gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/FeatureSupportTest.java

```diff
@@ -121,7 +121,8 @@ public void shouldSupportComputerIfAGraphCanCompute() throws Exception {
          */
         @Test
         @FeatureRequirement(featureClass = GraphFeatures.class, feature = FEATURE_TRANSACTIONS, supported = false)
-        public void shouldSupportTransactionsIfAGraphConstructsATx() throws Exception {
+        @FeatureRequirement(featureClass = GraphFeatures.class, feature = FEATURE_THREADED_TRANSACTIONS, supported = false)
+        public void shouldSupportTransactionsIfAGraphConstructsATx() {
             try {
                 graph.tx();
                 fail(String.format(INVALID_FEATURE_SPECIFICATION, GraphFeatures.class.getSimpleName(), FEATURE_TRANSACTIONS));
```

Comment:
Should `FEATURE_TRANSACTIONS` be updated here to `FEATURE_THREADED_TRANSACTIONS`?

Reply:
no, `FEATURE_THREADED_TRANSACTIONS` means transaction can be executed across multiple threads